main.go

```diff
@@ -119,6 +123,9 @@ func main() {
 			enry.GetLanguageType(language) != enry.Markup {
 			return nil
 		}
+		if *progLangs && enry.GetLanguageType(language) != enry.Programming {
```

Comment:
does it make sense to move this if before we check markup?

Reply:
It would probably be small performance improvement but knowing that `GetLanguageType` is just map lookup probably not much